application/backend/src/main/resources/apps/typerefinery/components/widgets/map/dialog/.content.json

```diff
@@ -5,13 +5,13 @@
     "configTab": {
```

Comment:
why the name and key is different here ?

Reply:
Done